### PR TITLE
159 disable modals background

### PR DESCRIPTION
**Pull request by VLegakis**
This blurs and disables interactions with portal's background content

### CODE REVIEW

File: src/components/icon/assets/node_slave.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (Azein) [inline src/components/icon/assets/node_slave.svg]: Are you sure it's not already added? As "node" or something else? I'm almost sure we are using this already, you can cycle through existing ones in storybook.

Comment (VLegakis) [reply src/components/icon/assets/node_slave.svg]: Let me recheck! Maybe I missed that this already exists

Comment (VLegakis) [reply src/components/icon/assets/node_slave.svg]: Yeap! It existed. I removed it! Thanks @Azein

**Comment by VLegakis** [comment]
> Current implementation isn't working. Layouting is wrong, as overlay blurs the content of portal, and also prevents any click events (because of pointer-events set to none), which we don't want for the portaled sidebar.

Indeed I had implemented the other way round by mistake! Disabling the portal's content instead of its background! Got confused with the right-left side somehow :( fixed now. Please take a look

**Pull request merged**